src/plugins/embeddable/public/tests/apply_filter_action.test.ts

```diff
@@ -95,26 +94,16 @@ test('ApplyFilterAction applies the filter to the root of the container tree', a
 });
 
 test('ApplyFilterAction is incompatible if the root container does not accept a filter as input', async () => {
-  const { doStart, coreStart, setup } = testPlugin();
-  const inspector = inspectorPluginMock.createStartContract();
+  const { doStart, setup } = testPlugin();
 
   const factory = new FilterableEmbeddableFactory();
   setup.registerEmbeddableFactory(factory.type, factory);
   const api = doStart();
   const applyFilterAction = createFilterAction();
-  const parent = new HelloWorldContainer(
-    { id: 'root', panels: {} },
-    {
-      getActions: () => Promise.resolve([]),
-      getEmbeddableFactory: api.getEmbeddableFactory,
-      getAllEmbeddableFactories: api.getEmbeddableFactories,
-      overlays: coreStart.overlays,
-      notifications: coreStart.notifications,
-      application: coreStart.application,
-      inspector,
-      SavedObjectFinder: () => null,
-    }
-  );
+
+  const parent = new HelloWorldContainer({ id: 'root', panels: {} }, {
+    getEmbeddableFactory: api.getEmbeddableFactory,
+  } as any);
```

Comment:
`as any` cast needed?  I know this pattern is scattered about but I think we should try to avoid casting to `any` in tests so prolifically as we do now.